actions/global_actions.jsx

```diff
@@ -264,11 +264,16 @@ export function toggleSideBarRightMenuAction() {
     };
 }
 
-export function emitBrowserFocus(focus) {
-    dispatch({
-        type: ActionTypes.BROWSER_CHANGE_FOCUS,
-        focus,
-    });
+export function browserChangedFocus(focus) {
```

Comment:
@sudheerDev @jespino I am not sure if this used by anything.

Reply:
Actually it is used by `logged_in.jsx`

Reply:
logged_in.jsx sets the value but I am not sure if anything consumes it.

Reply:
We use the corresponding reducer state in `notification_actions.jsx` as `state.views.browser.focused`